splice_si_api/src/main/java/com/splicemachine/si/impl/SimpleTxnFilter.java

```diff
@@ -89,13 +89,18 @@ public SimpleTxnFilter(String tableName,
                            TxnSupplier baseSupplier,
                            boolean ignoreNewerTransactions) {
         assert readResolver!=null;
-        this.transactionStore = new ActiveTxnCacheSupplier(baseSupplier, 100); //TODO -sf- configure
         this.myTxn=myTxn;
         this.readResolver=readResolver;
         this.ignoreNewerTransactions = ignoreNewerTransactions;
         SIDriver driver = SIDriver.driver();
-        if (driver != null) {
+        if (driver == null) {
+            // only happens during testing
+            this.transactionStore = new ActiveTxnCacheSupplier(baseSupplier, 128, 2048);
```

Comment:
If this only happens during testing, do we have a way to throw an exception if we're not in test mode?

Reply:
I couldn't find a good way to do that.